data/SpeakerPack/_template.md.j2

```diff
@@ -0,0 +1,174 @@
+- **10 minutes before** your talk, the green room AV team will check you are ready. If you are not, your talk might get skipped!
+{%- if room == 'curlyboi' %}
+
+The AV team will get in touch with you beforehand to provide more information on how your stream will be managed.
+
+{%- elif room == 'other' %}
+- **5 minutes before** your talk, a "virtual curtain" will drop, and you'll be given a link to connect to the studio. When you open that link, you'll be able to see and hear the stream return video feed, and the **backstage chat**.
+- **1 minute before** your talk, the virtual curtain will lift, and the MC will introduce you. (You'll see them and hear their voice through the stream return.)
+{% endif %}
+
+## During your talk
+
+Your talk will run for **15, 25, or 55 minutes**, depending on what you selected when you submitted your proposal.
+
+Your options for taking questions are:
+
+- Accept questions live on the stream. The MC will curate questions and read them to you. If you choose this option, question time is **included in your talk time**.
+- Discuss questions after the talk, via the "hallway" channels we provide (there will be text-only, audio and video options).
+- Decline to take questions.
+
+At the 5, 2, and 1 minute marks you'll get a countdown message in the backstage chat. **At the end of your allotted time** the MC will come back on stream, thank you for presenting, and the "virtual curtain" will drop.
+
+**Talks will not be allowed to run over**, so the MC may have to interrupt you if you don't wrap up in time; to avoid this, make sure you have the stream chat open!
+
+## After your talk
+
+As soon as the curtain drops you can leave the studio. (In fact, you'll be encouraged to rather quickly, because our changeover procedures are tight!) We'd love it if you could then go into one of the "hallway track" spaces we provide, but you don't have to.
+
+{% elif type == 'pre-record' %}
+## On the day
+
+On the day, **you'll need to be online in Venueless, our community platform** to answer questions during your talk. You don't need to go anywhere special&mdash;just join the room like an ordinary attendee.
+
+{% endif %}
+
+## Video uploads
+
+Some time after the event, if you've given us permission to, we'll upload the video from your talk to YouTube.
+
+## Conduct and expectations
+
+Please ensure your talk content and slides comply with [our code of conduct](/conduct). This includes your own behaviour as well as anything you endorse or encourage.
+
+In particular:
+
+- Don't include content where you attack or pentest conference networks, conference infrastructure, conference processes, or conference attendees, in any way.
+- If you are talking about past hacks or social engineering, please note clearly that you had informed consent for any system or person affected.
```

Comment:
I think, just as importantly, is actually obtaining that informed consent, and disclosing it.

Reply:
Good point!

Reply:
Oh oh - can you add something here about not playing copyrighted content (such as music)?
Somebody always does and YT gets grumpy.

Reply:
There's a checkbox for this on Pretalx :)